forem/forem
# API: Add instance endpoint and spec

## What type of PR is this? (check all applicable)

## Description
To power the upcoming Forem Directory, we need to add an endpoint to return data about the Forem instance.

## Related Tickets & Documents
N/A

## QA Instructions, Screenshots, Recordings
Run:

```bash
curl http://localhost:3000/api/instance
```

in a console window while your Rails server is running and check out the response 🎉 

### UI accessibility concerns?

N/A

## Added/updated tests?

## [Forem core team only] How will this change be communicated?

![lets_go_gif](https://media.giphy.com/media/RrVzUOXldFe8M/giphy.gif)
